src/printer.js

```diff
@@ -762,6 +762,14 @@ function genericPrintNoParens(path, options, print, args) {
       ]);
     }
     case "TSInterfaceDeclaration":
+      // Attach comments to the TSInterfaceBody since we want to print them
```

Comment:
Can you do that inside of `comments.js`, we do this kind of things a lot where we attach comments to different nodes